store/postgres/src/transaction_receipt.rs

```diff
@@ -1,72 +1,26 @@
 use diesel::{
-    pg::{Pg, PgConnection},
+    pg::PgConnection,
     prelude::*,
-    query_builder::{Query, QueryFragment},
-    sql_types::{Binary, Nullable},
+    sql_query,
+    sql_types::{Binary, Nullable, Text},
 };
-use diesel_derives::{Queryable, QueryableByName};
+use diesel_derives::QueryableByName;
 use graph::prelude::web3::types::H256;
 use itertools::Itertools;
 use std::convert::TryFrom;
 
 use graph::prelude::transaction_receipt::LightTransactionReceipt;
 
+use crate::ETHEREUM_BLOCKS_TABLE_NAME;
+
 /// Queries the database for all the transaction receipts in a given block range.
 pub fn find_transaction_receipts_in_block(
     conn: &PgConnection,
     blocks_table_name: &str,
     block_hash: H256,
 ) -> anyhow::Result<Vec<LightTransactionReceipt>> {
```

Comment:
Diffing got weird here, but I remove all the custom impls for `QueryFragment` and used  the`sql_query` funciton instead.